docs/resource-tuning-recommendations.md

```diff
@@ -165,14 +176,128 @@ A test was performed to collect the startup time of Jenkins under various resour
 | 1000m-req-2000m-limit-128m |                      131 |
 | 1000m-req-2000m-limit-512m |                       73 |
 
-The observations from the testing can be summarized as follows: 
-- CPU Limit has the largest affect on Startup Performance
-- CPU Request has little affect on Startup Performance
+The observations from the testing can be summarized as follows:
+
+- CPU Limit has the largest effect on Startup Performance
+- CPU Request has little effect on Startup Performance
 - The gain from a CPU Limit of 500m to 1000m is major
 - The gain from a CPU Limit of 1000m to 2000m is minor
-- One ideal configuration looks like:
-    - CPU Request: 100m
-    - CPU Limit: 1000m+
-    - Memory Request: 512M
-    - Memory Limit: 1-2GB (May vary depending on usage)
+- One ideal configuration looks like this:
+  - CPU Request: 100m
+  - CPU Limit: 1000m+
+  - Memory Request: 512M
+  - Memory Limit: 1-2GB (May vary depending on usage)
+
+### Advanced Jenkins Resource Tuning
+
+Consider monitoring the upper and lower bounds of CPU and memory usage of Jenkins instances over time. When idle, it has been observed that Jenkins uses under `5m` of CPU and about `650Mi` of memory. As per the **General Guidelines** above, "set requests to the *minimum* of what your application needs." It is ideal to reserve resources conservatively (especially for workloads that are often idle), and leverage resource limits to burst when active.
+
+![Jenkins CPU usage](assets/../images/jenkins-cpu-usage.png)
+
+Also, consider other workloads you may need to run in the tools namespace when accounting for requests/limits allocation to be within the allotted maximums.
+
+## Tools Namespaces Resource Quota Recommendations
+
+Every product in a cluster is provided a GUID and a namespace for each environment (i.e., dev, test, prod). These products also have a **tools** namespace defined as `<guid>-tools`, where tooling such as Jenkins are deployed.
```

Comment:
```suggestion
Every product in a cluster is provided a licence plate and a namespace for each environment (i.e., dev, test, prod). These products also have a **tools** namespace defined as `<license>-tools`, where tooling such as Jenkins are deployed.
```

I think in most places we refer to this bit as the license plate, not GUID. @mitovskaol can you confirm?

Reply:
Confirming that the correct way to refer to the project identifier must be a license plate.